BodyText/BodyText.js

```diff
@@ -118,11 +129,13 @@ const BodyTextBase = kind({
 			);
 		}
 		return (
-			<UiBodyText
-				{...rest}
-				centered={centered}
-				css={css}
-			/>
+			<Scroller verticalScrollbar="visible" focusableScrollbar={focusableScrollbar}>
```

Comment:
the scroller should not be visible all the time

Reply:
After refactoring the Scroller it will not be, as it won't have the buttons and you will only see the scrollTrack when content should be scrolled. If I change to verticalScrollbar="auto", the scrollbar won't take up its designated space.

Reply:
Alright. Thanks!

Reply:
I don't think we should wrap the BodyText with Scroller by default. if needed, The app will put in a scroller .
Everything in the GUI doesn't have to be provided by the component  side. Agate provides just the parts that can assembles.